VMBackup/main/fsfreezer.py

```diff
@@ -137,20 +137,28 @@ def freeze_safe(self,timeout):
         self.root_seen = False
         error_msg=''
         timedout = False
+        self.skip_freeze = True 
         try:
             freeze_result = FreezeResult()
             freezebin=os.path.join(os.getcwd(),os.path.dirname(__file__),"safefreeze/bin/safefreeze")
             args=[freezebin,str(timeout)]
-            arg=[]
+            no_mount_found = True
             for mount in self.mounts.mounts:
                 self.logger.log("fsfreeze mount :" + str(mount.mount_point), True)
                 if(mount.mount_point == '/'):
                     self.root_seen = True
                     self.root_mount = mount
                 elif(mount.mount_point and not self.should_skip(mount)):
+                    if(self.skip_freeze == True):
+                        self.skip_freeze = False
                     args.append(str(mount.mount_point))
-            if(self.root_seen):
+            if(self.root_seen and not self.should_skip(self.root_mount)):
+                if(self.skip_freeze == True):
+                    self.skip_freeze = False
```

Comment:
if there is no special handling for root mount, then we can remove "if(mount.mount_point == '/'):" and handle everything in single for loop

Reply:
yes.. ideally the mount output should parse root at the end. but this might be helpful in some cases. Lets put some telemetry first where root is not parsed at the end and then revert this code. For now lets keep it this way.